### PR TITLE
Respect interface settings

### PR DESCRIPTION
**Pull request by Laiff**
Closes: GTNewHorizons/GT-New-Horizons-Modpack#15117

### CODE REVIEW

**Review by MCTBL — APPROVED** [review]
fixed

**Review by OneEyeMaker — APPROVED** [review]

File: src/main/java/appeng/container/implementations/ContainerInterfaceTerminal.java [hunk 1]
```diff
@@ -312,8 +314,8 @@ private PacketInterfaceTerminalUpdate updateList() {
                             .setReps(machine.getSelfRep(), machine.getDisplayRep());
                     tracked.put(machine, entry);
                     trackedById.put(entry.id, entry);
+                    visited.add(machine);
                 }
-                visited.add(machine);
```
Comment (repo-alt) [inline src/main/java/appeng/container/implementations/ContainerInterfaceTerminal.java:316]: why is that necessary,  wouldn't 'continue' skip adding anyway?

Comment (Laiff) [reply src/main/java/appeng/container/implementations/ContainerInterfaceTerminal.java:316]: It it will produce unnecessary `remove` command better to do not try to remove element that are not present in terminal
